api/src/main/java/com/epam/esm/web/model/ErrorCode.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.web.model;
+
+/**
+ * Class-holder of error messages
+ */
+public class ErrorCode {
```

Comment:
Perhaps, it makes sense to document the fact that this class is not designed to be extended (for readability). But in fact, this class can't have sublasses anyway, since it has a private constructor.